project-keeper/src/test/java/com/exasol/projectkeeper/validators/pom/PomFileGeneratorTest.java

```diff
@@ -20,12 +20,28 @@
 import org.junit.jupiter.params.provider.MethodSource;
 
 import com.exasol.projectkeeper.ProjectKeeperModule;
+import com.exasol.projectkeeper.config.ProjectKeeperConfig;
 
 class PomFileGeneratorTest {
 
+    static Stream<Arguments> testGenerationWithParentPomCases() {
+        return Stream.of(//
+                Arguments.of(null, "../pom.xml" /* that seems to be a default in the reader */),
+                Arguments.of("./myParentPom.xml", "./myParentPom.xml")//
+        );
```

Comment:
When using only simple types (strings, int, ...) you could use the `@CsvSource` annotation and specify the parameters directly at the test.

Reply:
I had that in the beginning but the null was not very clear in that notation

Reply:
👍